docs/syntax.md

```diff
@@ -445,28 +460,39 @@ i24 = -42
 u24 = 42
 ```
 
-| Operation      | Syntax |
-| -------------- | ------ |
-| Addition       | x + y  |
-| Subtraction    | x - y  |
-| Multiplication | x \* y |
-| Division       | x / y  |
-| Equal          | x == y |
-| Not Equal      | x != y |
-| Less Than      | x < y  |
-| Greater Than   | x > y  |
-| Bitwise And    | x & y  |
-| Bitwise Or     | x \| y |
-| Bitwise Xor    | x ^ y  |
+| Operation      | Syntax   |
+| -------------- | -------- |
```

Comment:
Here and below on the other syntax, we need to explain that the three number types cannot be mixed and list what types can use which operation in the table